docker_explorer/storage.py

```diff
@@ -159,7 +142,8 @@ def MakeMountCommands(self, container_object, mount_dir):
              'ro,remount,append:{0:s}=ro+wh'.format(mountpoint_path), 'none',
              mount_dir])
 
-    commands.extend(self._MakeExtraVolumeCommands(container_object, mount_dir))
+    # Adding the commands to mount any extra declared Volumes and Mounts
+    commands.extend(self._MakeVolumeMountCommands(container_object, mount_dir))
```

Comment:
I believe the return type in this method is list of lists not list of strings, please look into the doc-string of this method and change as needed.

Reply:
correct, fixed